src/Raven.Server/Documents/TcpHandlers/SubscriptionDocumentsFetcher.cs

```diff
@@ -34,7 +35,7 @@ public class SubscriptionDocumentsFetcher
         private readonly SubscriptionState _subscription;
         private readonly SubscriptionPatchDocument _patch;
         private NativeMemory.ThreadStats _threadAllocations;
-        private Size _currentMaximumAllowedMemory = new Size(32, SizeUnit.Megabytes);
+        private Size _MaximumAllowedMemory;
```

Comment:
`_MaximumAllowedMemory` -> `_maximumAllowedMemory`

Need to add a comment explaining why we have this limit

Reply:
done